core/src/main/java/org/mobilitydata/gtfsvalidator/notice/NonAsciiOrNonPrintableCharNotice.java

```diff
@@ -20,12 +20,14 @@
 
 /** ID value contains something different from printable ASCII characters. */
```

Comment:
Should have a similar comment header as other notices with more description?

Reply:
What description do you want me to put here?

Please note that many notices (such as EmptyFileNotice) have one-line comments.

Reply:
To match the other data type errors, I'd add this from the "ID" definition in GTFS:

>An ID field value is an internal ID, not intended to be shown to riders, and is a sequence of any UTF-8 characters. Using only printable ASCII characters is recommended.

Reply:
Good point. Done.